wazo_agentd/http.py

```diff
@@ -178,21 +156,33 @@ def get(self, agent_number):
 class _LoginAgentById(_BaseResource):
     @required_acl('agentd.agents.by-id.{agent_id}.login.create')
     def post(self, agent_id):
-        extension, context = _extract_extension_and_context()
+        body = agent_login_schema.load(request.get_json(force=True))
         tenant_uuids = self._build_tenant_list({'recurse': True})
         self.service_proxy.login_agent_by_id(
-            agent_id, extension, context, tenant_uuids=tenant_uuids
+            agent_id, body['extension'], body['context'], tenant_uuids=tenant_uuids
         )
         return '', 204
 
 
 class _LoginAgentByNumber(_BaseResource):
     @required_acl('agentd.agents.by-number.{agent_number}.login.create')
     def post(self, agent_number):
-        extension, context = _extract_extension_and_context()
+        body = agent_login_schema.load(request.get_json(force=True))
```

Comment:
No need to add force=True
BaseSchema already handle this [here](https://github.com/wazo-platform/xivo-lib-python/blob/master/xivo/mallow_helpers.py#L82)

Reply:
It's fixed. Thank you!